stl/inc/format

```diff
@@ -1985,6 +2103,109 @@ _NODISCARD _OutputIt _Fmt_write(
     return _Fmt_write(_STD move(_Out), basic_string_view<_CharT>{_Value}, _Specs, _Locale);
 }
 
+inline constexpr char16_t _U16_width_estimate_bounds[] = { // Per N4885 [format.string.std]/11
+    0x1100u, 0x1160u, 0x2329u, 0x232Bu, 0x2E80u, 0x303Fu, 0x3040u, 0xA4D0u, 0xAC00u, 0xD7A4u, 0xF900u, 0xFB00u, 0xFE10u,
+    0xFE1Au, 0xFE30u, 0xFE70u, 0xFF00u, 0xFF61u, 0xFFE0u, 0xFFE7u};
+
+inline constexpr char32_t _U20_width_estimate_bounds[] = { // Per N4885 [format.string.std]/11

```

Comment:
Why "U20"?

Reply:
"UNN" in these names is "U" for Unicode and NN meaning the values in this table can be represented in NN bits. In no way do I claim these are great or even good names, I'm simply stating the rationale. I think `_Width_estimate_high_intervals` (respectively `low_intervals`) is a bit better? I'll make that change, but feel free to suggest anything you prefer.